packages/block-library/src/columns/style.scss

```diff
@@ -85,8 +81,8 @@
 		> .wp-block-column {
 			// Available space should be divided equally amongst columns.
 			flex-basis: 0;
-			flex-grow: 1;
```

Comment:
I don't know why this changed, I didn't touch this piece. May have been tweaked by a linter.